<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationDialog</class>
 <widget class="QDialog" name="InformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>978</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cihaz Kimlik</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QGroupBox" name="ModelBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_19">
            <item>
             <widget class="QLabel" name="Model_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">verticalLayout_6 {
border:1px;
}</string>
              </property>
              <property name="text">
               <string>Model</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="model_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="CihazSeriNoBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_20">
            <item>
             <widget class="QLabel" name="cihazSeriNo_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">verticalLayout_6 {
border:1px;
}</string>
              </property>
              <property name="text">
               <string>Cihaz Seri No</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cihaz_seri_no_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="UIDBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_21">
            <item>
             <widget class="QLabel" name="UIDNo_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>UID No</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="UIDNo_1">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="TestDurumBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_22">
            <item>
             <widget class="QLabel" name="testDurumu_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">verticalLayout_6 {
border:1px;
}</string>
              </property>
              <property name="text">
               <string>Test Durumu</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="test_durum_">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="uretimTarihBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_23">
            <item>
             <widget class="QLabel" name="uretim_tarih_2">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Ãœretim Tarihi</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>1</number>
              </property>
              <item>
               <widget class="QSpinBox" name="date_day">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="date_month">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="date_year">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_25">
         <item>
          <widget class="QGroupBox" name="AnakartNoBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_26">
            <item>
             <widget class="QLabel" name="anakart_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>AnakartNo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="anakart_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="ModemKartBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_27">
            <item>
             <widget class="QLabel" name="modem_karti_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Modem KartÄ±</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="modem_karti_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="DurumBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_28">
            <item>
             <widget class="QLabel" name="Durum_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">verticalLayout_6 {
border:1px;
}</string>
              </property>
              <property name="text">
               <string>Durum</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="durum_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="SarjKartBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_29">
            <item>
             <widget class="QLabel" name="sarj_karti_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Åžarj KartÄ±</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sarj_karti_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="LcdKartBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_30">
            <item>
             <widget class="QLabel" name="lcd_karti_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>LCD Kart</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="lcd_karti_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="modemSeriNoBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_31">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Modem Seri NumaralarÄ±</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="0">
             <widget class="QLabel" name="modemSeri3_2">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Modem Seri 3</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="modemSeri5_2">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Modem Seri 5</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="modemSeri2_1">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="modemSeri4_1">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="modemSeri5_1">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="modemSeri1_2">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Modem Seri 1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="modemSeri4_2">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Modem Seri 4</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="modemSeri2_2">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Modem Seri 2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="modemSeri1_1">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="modemSeri3_1">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="modemSeri6_2">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Modem Seri 6</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="modemSeri6_1">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_32">
         <item>
          <widget class="QGroupBox" name="NotlarBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_33">
            <item>
             <widget class="QLabel" name="notlar_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Notlar</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="notlar_1"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">/*#pushButton {
color:white;
background-color:rgb(166, 255, 146);
border-width:0px;
border-radius: 4px;
}

#pushButton:hover{

background-color:rgb(23, 139, 77)

}
*/</string>
           </property>
           <property name="text">
            <string>GÃ¼ncelle</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
